components/bl-list-component/src/list-content.js

```diff
@@ -0,0 +1,53 @@
+import { List } from './list';
+
+export function ListContent({
+  component, eventHandlers, child}) {
+
+  const { color, fontSize, listItems, type} = component;
+
+  const itemStyles = {
+    color   : color,
+    fontSize: fontSize
+  };
+
+  const onItemClick = (key) => {
+    eventHandlers.onClickListItem({item: key});
+  }
+
+ const getItems = (items) => {
+    return (items || []).map((item, i) => {
+      if (item.child && item.child.length !== 0) {
+          return (
+            <li key={i.toString()}
+              className="list__item"
+              style = {itemStyles}
+              onClick={(e) => {
+                e.stopPropagation();
+                onItemClick(item.item);
+              }}
+            >
+              {item.item}
+              <List component={component}
+                    child={item.child}
+                    eventHandlers={eventHandlers}/>
+            </li>
+          )
+      } else {
+        return (
+          <li key={i.toString()}
+            className="list__item"
+            style = {itemStyles}
+            onClick={(e) => {
+              e.stopPropagation();
+              onItemClick(item.item);
+            }}
+          >
+            {item.item}
+          </li>
+        )
+      }
```

Comment:
That should gone after other changes, but you was able short it twice

```suggestion
          return (
            <li key={i}
              className="list__item"
              style={itemStyles}
              onClick={(e) => onItemClick(e, item) }
            >
              { item.content }
              { item.child && (
                  <List component={component}
                    child={item.child}
                    eventHandlers={eventHandlers}
                  />
               ) }
            </li>
          )
      }
```